workflow/common/common.go

```diff
@@ -260,6 +260,9 @@ const (
 	ArgoProgressPath = VarRunArgoPath + "/progress"
 
 	ConfigMapName = "workflow-controller-configmap"
+
+	// DefaultPageSize is the limit value of the request k8s apiserver
```

Comment:
```suggestion
	// DefaultPageSize is the limit value of the request to the k8s apiserver. Aligned with client-go default: https://github.com/kubernetes/client-go/blob/ee1a5aaf793a9ace9c433f5fb26a19058ed5f37c/tools/pager/pager.go#L31
```

Let's put your PR comment into the in-line comment